templates/desktop/features.html

```diff
@@ -1,384 +1,586 @@
       <h2>Organise your photos</h2>
-      <p>With Shotwell, you can quickly and easily import, organise, edit and view your pictures. And you can share your favourite snaps on all popular photo sites and social networks.</p>
     </div>
-    <div class="col-4 p-divider__block">
+    <div class="col">
+      <p>With Shotwell, you can quickly and easily import, organise, edit and view your pictures. And you can share your
+        favourite snaps on all popular photo sites and social networks.</p>
+    </div>
+  </div>
+  <div class="row--25-75">
+    <div class="col">
+      {{
+      image(
+      url="https://assets.ubuntu.com/v1/1aef005e-desktop-22.04-photos.jpg",
+      alt="",
+      width="1360",
+      height="768",
+      hi_def=True,
+      loading="lazy",
+      ) | safe
+      }}
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
       <h2>Edit and illustrate</h2>
-      <p>Edit your photos or create professional illustrations and designs with tools like Gimp and Krita, available in the Ubuntu Software centre.</p>
-      <div class="p-logo-section has-misaligned-images">
-        <div class="p-logo-section__items">
-          <div class="p-logo-section__item">
+    </div>
+    <div class="col">
+      <p>Edit your photos or create professional illustrations and designs with tools like Gimp and Krita, available in
+        the Ubuntu Software centre.</p>
+      <ul class="p-inline-list u-no-margin--bottom" style="display: flex; align-items: center;">
+        <li class="p-inline-list__item">
+          <div class="u-inline--child">
             {{
-              image(
-                url="https://assets.ubuntu.com/v1/b38df947-krita.png",
-                alt="Krita icon",
-                height="60",
-                width="60",
-                hi_def=True,
-                attrs={"class": "p-logo-section__logo"},
-                loading="lazy"
-              ) | safe
+            image(
+            url="https://assets.ubuntu.com/v1/b38df947-krita.png",
+            alt="Krita icon",
+            width="80",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
             }}
           </div>
-          <div class="p-logo-section__item">
+        </li>
+        <li class="p-inline-list__item">
+          <div class="u-inline--child">
             {{
-              image(
-                url="https://assets.ubuntu.com/v1/4d47a674-gimp-60x43.svg",
-                alt="",
-                height="60",
-                width="60",
-                hi_def=True,
-                loading="lazy",
-                attrs={"class": "p-logo-section__logo"},
-              ) | safe
+            image(
+            url="https://assets.ubuntu.com/v1/4d47a674-gimp-60x43.svg",
+            alt="Gimp icon",
+            width="80",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
             }}
```

Comment:
Indentation

Reply:
This has been done automatically by the formatter